stable/jenkins/values.yaml

```diff
@@ -18,8 +18,10 @@ Master:
   UseSecurity: true
   AdminUser: admin
   # AdminPassword: <defaults to random>
-  Cpu: "200m"
-  Memory: "256Mi"
+  Cpu: "2000m"
+  Memory: "2048Mi"
+  CpuLimit: "2000m"
+  MemoryLimit: "2048Mi"
```

Comment:
Our best practice is to not specify defaults for resources at all. It should be possible to install and test charts on clusters with little resources, such as Minikube.